transactionoutbox-core/src/test/java/com/gruelbox/transactionoutbox/TestDefaultPersistorConfiguration.java

```diff
@@ -35,6 +38,30 @@ final void whenMigrateIsFalseDoNotMigrate() throws Exception {
         });
   }
 
+  @Test
+  final void writeSchema() {
```

Comment:
I was not sure where to add this test.

If it is added to `AbstractDefaultPersistorTest`, then the output could be asserted for each dialect. Maybe a bit much?

Also, it does not really feel like `TestDefaultPersistorConfiguration` is the best place.

However, added a new `DefaultPersistorTest` class may just be confusing.